akka-persistence/src/test/scala/akka/persistence/SnapshotFailureRobustnessSpec.scala

```diff
@@ -97,7 +97,12 @@ object SnapshotFailureRobustnessSpec {
 class SnapshotFailureRobustnessSpec extends PersistenceSpec(PersistenceSpec.config("leveldb", "SnapshotFailureRobustnessSpec", serialization = "off", extraConfig = Some(
   """
   akka.persistence.snapshot-store.local.class = "akka.persistence.SnapshotFailureRobustnessSpec$FailingLocalSnapshotStore"
-  akka.persistence.snapshot-store.local-delete-fail.class = "akka.persistence.SnapshotFailureRobustnessSpec$DeleteFailingLocalSnapshotStore"
+  akka.persistence.snapshot-store.local-delete-fail {
+    class = "akka.persistence.SnapshotFailureRobustnessSpec$DeleteFailingLocalSnapshotStore"
+    stream-dispatcher = "akka.persistence.dispatchers.default-stream-dispatcher"
+    dir = "target/snapshots-SnapshotFailureRobustnessSpec/"
+    max-load-attempts = 3
```

Comment:
_max-load-attempts_ is a required key. If we do not mention it, config throws 
`com.typesafe.config.ConfigException$Missing: No configuration setting found for key 'max-load-attempts'` 
other keys are also required. User must specify them if they want different config path